docs/output.rst

```diff
@@ -5,12 +5,13 @@ Outputs
 This page explains the output file formats for equilibrium solutions
 computed by DESC.
 
-ASCII
+HDF5
 ************
 
-.. include:: ./text_output_format.rst
+.. include:: ./binary_output_format.rst
 
-HDF5
+
+ASCII
```

Comment:
could also add a note here recommending HDF5 instead, or possibly add a note to the docstring itself